common/src/main/java/com/microsoft/identity/common/internal/ui/browser/BrowserAuthorizationStrategy.java

```diff
@@ -95,15 +95,16 @@ public BrowserAuthorizationStrategy(@NonNull Activity activity, @NonNull Intent
         final Uri requestUrl = authorizationRequest.getAuthorizationRequestAsHttpRequest();
         authIntent.setData(requestUrl);
 
-        mReferencedActivity.get().startActivity(
-                AuthorizationActivity.createStartIntent(
-                        mReferencedActivity.get().getApplicationContext(),
-                        authIntent,
-                        mResultIntent,
-                        requestUrl.toString(),
-                        mAuthorizationRequest.getRedirectUri(),
-                        mAuthorizationRequest.getRequestHeaders(),
-                        AuthorizationAgent.BROWSER));
+        final Intent intent = AuthorizationActivity.createStartIntent(
+                mReferencedActivity.get().getApplicationContext(),
+                authIntent,
+                mResultIntent,
+                requestUrl.toString(),
+                mAuthorizationRequest.getRedirectUri(),
+                mAuthorizationRequest.getRequestHeaders(),
+                AuthorizationAgent.BROWSER);
+        intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
```

Comment:
Could you add a comment on this? like " singleTask launchMode is required for the authorization redirect in the browser authorization flow". Thanks.

Reply:
+1 This feels easy for someone months or years down the line to inadvertently break